.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -47,11 +85,7 @@ jobs:
 
     strategy:
       matrix:
-        include:
-          - arch: amd64
-            os: windows-latest
-          - arch: arm64
-            os: cirun-win11-23h2-pro-arm64-16-2024-04-22
```

Comment:
Why not use the same form as used in the other repo? Does it not work?

Reply:
No because in those repos we're not specifying the runners using a matrix - we're just choosing to run on either the windows-latest or the cirun vms but not both, as we are here. :/ Github doesn't allow conditional matrix entries, and the semantics of `jobs.<job-id>.if` and `jobs.<job-id>.strategy.matrix.exclude` don't provide a simpler solution for matrix exclusion.

Reply:
I mean:
```
- arch: arm64
  os: "${{ vars.USE_CIRUN == 'true' && format('cirun-win11-23h2-pro-arm64-64-2024-04-22--{0}', github.run_id) || 'windows-latest' }}
```

This doesn't work? That's too bad if it's the case.

Reply:
Ah I see. Yeah that won't work because it'd attempt to install the arm64 Swift toolchain on the x64 windows-latest image which doesn't work from my last attempts... but now I'm second guessing myself. Let me try again.

Reply:
Ah ok it makes sense. Then I'd recommend to use https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
Sadly, that doesn't work because of:
> Note: The jobs.<job_id>.if condition is evaluated before [jobs.><job_id>.strategy.matrix](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) is applied.

Which means we can't write something like:

```
if: ${{ matrix.is_cirun == "false" || vars.USE_CIRUN }}
```

Reply:
sad panda